test/integration/good/function-signatures/distributions/univariate/discrete/discrete_range/pretty.expected

```diff
@@ -0,0 +1,158 @@
+  transformed_data_real = discrete_range_cdf(d_int_array, d_int_array,
+                                             d_int_array);
+}
+parameters {
+  real y_p;
+}
+transformed parameters {
+  real transformed_param_real;
+  transformed_param_real = discrete_range_cdf(d_int, d_int, d_int);
+  transformed_param_real = discrete_range_cdf(d_int, d_int, d_int_array);
+  transformed_param_real = discrete_range_cdf(d_int, d_int_array, d_int);
+  transformed_param_real = discrete_range_cdf(d_int, d_int_array,
+                                              d_int_array);
+  transformed_param_real = discrete_range_cdf(d_int_array, d_int, d_int);
+  transformed_param_real = discrete_range_cdf(d_int_array, d_int,
+                                              d_int_array);
+  transformed_param_real = discrete_range_cdf(d_int_array, d_int_array,
+                                              d_int);
+  transformed_param_real = discrete_range_cdf(d_int_array, d_int_array,
+                                              d_int_array);
+}
+model {
+  y_p ~ normal(0, 1);
+}
+
+  $ ../../../../../../../../../install/default/bin/stanc --auto-format discrete_range_lccdf.stan
+data {
+  int d_int;
+  array[d_int] int d_int_array;
+}
+transformed data {
+  real transformed_data_real;
+  transformed_data_real = discrete_range_lccdf(d_int, d_int, d_int);
+  transformed_data_real = discrete_range_lccdf(d_int, d_int, d_int_array);
+  transformed_data_real = discrete_range_lccdf(d_int, d_int_array, d_int);
+  transformed_data_real = discrete_range_lccdf(d_int, d_int_array,
+                                               d_int_array);
+  transformed_data_real = discrete_range_lccdf(d_int_array, d_int, d_int);
+  transformed_data_real = discrete_range_lccdf(d_int_array, d_int,
+                                               d_int_array);
+  transformed_data_real = discrete_range_lccdf(d_int_array, d_int_array,
+                                               d_int);
+  transformed_data_real = discrete_range_lccdf(d_int_array, d_int_array,
+                                               d_int_array);
+}
+parameters {
+  real y_p;
+}
+transformed parameters {
+  real transformed_param_real;
+  transformed_param_real = discrete_range_lccdf(d_int| d_int, d_int);
+  transformed_param_real = discrete_range_lccdf(d_int| d_int, d_int_array);
+  transformed_param_real = discrete_range_lccdf(d_int| d_int_array, d_int);
+  transformed_param_real = discrete_range_lccdf(d_int| d_int_array, d_int_array);
+  transformed_param_real = discrete_range_lccdf(d_int_array| d_int, d_int);
+  transformed_param_real = discrete_range_lccdf(d_int_array| d_int, d_int_array);
+  transformed_param_real = discrete_range_lccdf(d_int_array| d_int_array, d_int);
+  transformed_param_real = discrete_range_lccdf(d_int_array| d_int_array, d_int_array);
+}
+model {
+  y_p ~ normal(0, 1);
+}
+
+
+Semantic error in 'discrete_range_lccdf.stan', line 7, column 26 to column 67:
```

Comment:
Is this an actual error?

Reply:
Yes it is. Let me check.

Reply:
This was legit, I did not use vertical bars for lccdf calls in trans. data.